### PR TITLE
Change default values for Bitnami Postgres helm chart 

### PR DESCRIPTION
**Pull request by sturman**
## About the changes

Bitnami PostgreSQL helm charts does not have `postgresqlDatabase`, `postgresqlUsername` and `postgresqlPassword` so the keys were changed.

### CODE REVIEW

**Comment by gastonfournier** [comment]
Hi @sturman 
I'm happy to look into this, but I'm not familiar with how to verify that this change is sound. Would you mind adding some manual test steps to the description of the PR (or link to the documentation if there's any), so I can validate the change?
Thanks!

File: charts/unleash/values.yaml [hunk 1]
```diff
@@ -231,10 +231,11 @@ vpa:
...
   fullnameOverride: "unleash-postgresql"
-  postgresqlDatabase: unleash
-  postgresqlUsername: unleash
-  # this should be change to something more secure or use an existing secret
-  postgresqlPassword: unleash
+  auth:
+    username: unleash
+    # this should be change to something more secure or use an existing secret
+    password: unleash
+    database: unleash
```
Comment (sturman) [inline charts/unleash/values.yaml:238]: @gastonfournier here are values from Bitnami postgresql chart https://github.com/bitnami/charts/blob/1601a1b831d4c45dbc285ed86c4b83ff52fbeb1f/bitnami/postgresql/values.yaml#L121-L136

File: charts/unleash/templates/deployment.yaml [hunk 2]
```diff
@@ -36,7 +36,7 @@ spec:
               valueFrom:
                 secretKeyRef:
                   name: {{ if .Values.dbConfig.useExistingSecret.name }}{{ .Values.dbConfig.useExistingSecret.name }}{{ else }}{{ .Values.postgresql.fullnameOverride }}{{ end }}
-                  key: {{ if .Values.dbConfig.useExistingSecret.key }}{{ .Values.dbConfig.useExistingSecret.key }}{{ else }}postgres-password{{ end }}
+                  key: {{ if .Values.dbConfig.useExistingSecret.key }}{{ .Values.dbConfig.useExistingSecret.key }}{{ else }}password{{ end }}
```
Comment (sturman) [inline charts/unleash/templates/deployment.yaml:39]: @gastonfournier 
`postgres-password` is the Secret key where a password for `postgres` user is stored. https://github.com/bitnami/charts/blob/1601a1b831d4c45dbc285ed86c4b83ff52fbeb1f/bitnami/postgresql/templates/secrets.yaml#L17
The password for `unleash` user is stored in `password` key.

**Review by gastonfournier — APPROVED** [review]
> Also, the Unleash chart is successfully installed with default values in CI

Ok, I see the [build failed](https://github.com/Unleash/helm-charts/actions/runs/4344341516/jobs/7587590773#step:8:665) prior to your change https://github.com/Unleash/helm-charts/pull/71/commits/32d4658592bf3e42659e23ac865695f3dd878f8e 

I also checked the diff between the two versions (under https://github.com/bitnami/charts repo) `git diff f5c23f6a5379f3e3932c8cdc2c93a73505b41bc9 1601a1b831d4c45dbc285ed86c4b83ff52fbeb1f  -- bitnami/postgresql/values.yaml` and saw the change in the configuration parameters.

So, thanks for all your insights! I think overall this looks good and the CI passes so :+1:

**Comment by promiseofcake** [comment]
This fails on a default install locally. 

```bash
$ kubectl logs -f unleash-7778d47cc7-v2bxv                                                             1
[2023-03-10T17:57:32.537] [ERROR] server-impl.js - Failed to migrate db error: password authentication failed for user "unleash"
    at Parser.parseErrorMessage (/unleash/node_modules/pg-protocol/dist/parser.js:287:98)
    at Parser.handlePacket (/unleash/node_modules/pg-protocol/dist/parser.js:126:29)
    at Parser.parse (/unleash/node_modules/pg-protocol/dist/parser.js:39:38)
    at Socket.<anonymous> (/unleash/node_modules/pg-protocol/dist/index.js:11:42)
    at Socket.emit (node:events:513:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at Socket.Readable.push (node:internal/streams/readable:228:10)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
  length: 103,
  severity: 'FATAL',
  code: '28P01',
  detail: undefined,
  hint: undefined,
  position: undefined,
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'auth.c',
  line: '326',
  routine: 'auth_failed'
}
[ERROR] error: password authentication failed for user "unleash"
    at Parser.parseErrorMessage (/unleash/node_modules/pg-protocol/dist/parser.js:287:98)
    at Parser.handlePacket (/unleash/node_modules/pg-protocol/dist/parser.js:126:29)
    at Parser.parse (/unleash/node_modules/pg-protocol/dist/parser.js:39:38)
    at Socket.<anonymous> (/unleash/node_modules/pg-protocol/dist/index.js:11:42)
    at Socket.emit (node:events:513:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at Socket.Readable.push (node:internal/streams/readable:228:10)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
```

But so did the previous version.

**Comment by sturman** [comment]
@promiseofcake helm install with default values works for me. Here is the terminal output.
```
minikube (unleash) ~/projects/contribute 
❯ k get pods
No resources found in unleash namespace.

minikube (unleash) ~/projects/contribute 
❯ helm search repo unleash
NAME                    CHART VERSION   APP VERSION     DESCRIPTION                                       
unleash/unleash         2.7.7           4.19.4          Unleash is a open source feature flag & toggle ...
unleash/unleash-proxy   0.2.2           v0.13.1         A Helm chart for Kubernetes                       

minikube (unleash) ~/projects/contribute 
❯ helm install unleash unleash/unleash --version 2.7.7
NAME: unleash
LAST DEPLOYED: Sat Mar 11 10:51:38 2023
NAMESPACE: unleash
STATUS: deployed
REVISION: 1
NOTES:
1. Get the application URL by running these commands:
  export POD_NAME=$(kubectl get pods --namespace unleash -l "app.kubernetes.io/name=unleash,app.kubernetes.io/instance=unleash" -o jsonpath="{.items[0].metadata.name}")
  export CONTAINER_PORT=$(kubectl get pod --namespace unleash $POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}")
  echo "Visit http://127.0.0.1:8080 to use your application"
  kubectl --namespace unleash port-forward $POD_NAME 8080:$CONTAINER_PORT

minikube (unleash) ~/projects/contribute 
❯ k get pods -w
NAME                      READY   STATUS    RESTARTS   AGE
unleash-8cbdc6df6-kdw2z   0/1     Error     0          7s
unleash-postgresql-0      0/1     Running   0          7s
unleash-8cbdc6df6-kdw2z   0/1     Running   1          7s
unleash-8cbdc6df6-kdw2z   0/1     Error     1          9s
unleash-8cbdc6df6-kdw2z   0/1     CrashLoopBackOff   1          10s
unleash-postgresql-0      1/1     Running            0          15s
unleash-8cbdc6df6-kdw2z   0/1     Running            2          24s
unleash-8cbdc6df6-kdw2z   1/1     Running            2          100s
```

**Pull request merged**